README.md

```diff
@@ -1 +1,111 @@
-# README
+# README for Acute Myeloid Leukemia Heatmap Analysis
+Overview
+This repository contains an analysis of RNA-seq data related to Acute Myeloid Leukemia (AML). The analysis utilizes a sample dataset from Shih et al., 2017 and is designed to create clustered heatmaps to visualize gene expression patterns in AML.
```

Comment:
We could maybe add more information about where this analysis was done and who was involved.